python/semantic_kernel/connectors/ai/google/shared.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from semantic_kernel.contents.chat_history import ChatHistory
+from semantic_kernel.contents.utils.author_role import AuthorRole
+from semantic_kernel.exceptions.service_exceptions import ServiceInvalidRequestError
+
+
+def filter_system_message(chat_history: ChatHistory) -> str | None:
```

Comment:
nit: is there a better file name choice instead of something as generic as `shared.py`?

Reply:
I'm curious why move this from utils.py to shared.py?

Reply:
How about `shared_utils.py`?